config.toml.example

```diff
@@ -111,6 +111,29 @@
 # General build configuration options
 # =============================================================================
 [build]
+# The default stage to use for a compile.
+#
+# This is overridden by `doc-stage`, `build-stage`, etc.
+# Default: none
+#stage = "complicated"
```

Comment:
What is `"complicated"`? Shouldn't this be `"none"`?

Reply:
Yeah, that makes sense. I was thinking 'complicated' as in 'doc-stage and build-stage are different', but those are their own config options and this default to none.
```suggestion
#stage = "none"
```